src/vs/workbench/contrib/terminalContrib/links/browser/terminalLinkManager.ts

```diff
@@ -128,6 +128,8 @@ export class TerminalLinkManager extends DisposableStore {
 						}
 					]);
 				}
+				// This doesnt get called for the RESULT of OSC 8 hyperlink to folder.
```

Comment:
This part wasn't getting called for OSC 8 hyperlink, but did for directory path if followed by `echo` for example.